pkg/flowaggregator/flowaggregator.go

```diff
@@ -130,8 +138,16 @@ var (
 		"destinationServicePortName",
 		"ingressNetworkPolicyName",
 		"ingressNetworkPolicyNamespace",
+		"ingressNetworkPolicyUID",
+		"ingressNetworkPolicyType",
+		"ingressNetworkPolicyRuleName",
+		"ingressNetworkPolicyRulePriority",
```

Comment:
I believe go-ipfix has not supported correlation of data type `signed32`, we should add that before using it here.

Reply:
Not sure if I get it. I have already changed rule priority's type to signed32 in go-ipfix. In the output of template set, it shows rule priority's length to be 4.

Reply:
I get it.. Will make more correlated changes in go-ipfix